alshakerM/my-instagram
# Generate more posts and fix user avatars

### Changes

1. This PR creates a bigger data set for posts (120 posts now).
2. It updates the users' database (since it's derived from posts data).
3. Fixes avatars and usernames for users in comments and replies. Now they're all existing users (have profiles with posts).

### Testing
1. Verify that posts load successfully.
2. Verify that avatars in comments and replies have real people now.
3. Verify that usernames in comments and replies represent existing users (by copying the username and checking in /username).
